packages/common/src/sendTransaction.ts

```diff
@@ -38,10 +39,11 @@ export type SendTransactionExtraOptions<chain extends Chain | undefined> = {
 export async function sendTransaction<
   chain extends Chain | undefined,
   account extends Account | undefined,
-  chainOverride extends Chain | undefined,
+  const request extends SendTransactionRequest<chain, chainOverride>,
+  chainOverride extends Chain | undefined = undefined,
 >(
   client: Client<Transport, chain, account>,
-  request: SendTransactionParameters<chain, account, chainOverride>,
+  request: SendTransactionParameters<chain, account, chainOverride, request>,
```

Comment:
these changes align with viem's type signature (forgot to do this last version bump)